base/tomcat-9.0/conf/server.xml

```diff
@@ -24,7 +24,7 @@
      define subcomponents such as "Valves" at this level.
      Documentation at /docs/config/server.html
  -->
-<Server port="[pki_tomcat_server_port]" shutdown="SHUTDOWN">
+<Server port="8005" shutdown="SHUTDOWN">
```

Comment:
Not sure I understand this part, why are you no longer using the `pki_tomcat_server_port` property here?

Reply:
@ckelleyRH The `server.xml` coming with tomcat has port 8005. I think @edewata goal is to remove all the customisation in this file so we can use the one bundled with tomcat and do the manipulation from python. So instead of a variable in the file, this used in the python code to update the file.

Reply:
OK, so the aim is to get to [this](https://github.com/apache/tomcat/blob/main/conf/server.xml) and not maintain a modified `server.xml` - makes sense, thanks!

Reply:
Right. This could make it easier to support multiple Tomcat versions at the same time.

Reply:
> Right. This could make it easier to support multiple Tomcat versions at the same time.

Good, because we might be about to do just this depending on how difficult it is to update `tomcatjss` to work with Tomcat 10.